tests/test_image_supervised.py

```diff
@@ -69,11 +69,43 @@ def test_timout(_):
     clf = ImageClassifier(path=path, verbose=False)
     train_x = np.random.rand(100, 25, 25, 1)
     train_y = np.random.randint(0, 5, 100)
-    with pytest.raises(TimeoutError) as e:
+    with pytest.raises(TimeoutError):
         clf.fit(train_x, train_y, time_limit=1)
     clean_dir(path)
 
 
+@patch('multiprocessing.Pool', new=MockProcess)
+@patch('autokeras.search.ModelTrainer.train_model', side_effect=mock_train)
+def test_timout_resume(_):
+    Constant.MAX_ITER_NUM = 1
+    # make it impossible to complete within 10sec
+    Constant.MAX_MODEL_NUM = 1000
+    Constant.SEARCH_MAX_ITER = 1
+    Constant.T_MIN = 0.8
```

Comment:
@tl-yang 
If the timeout is working, it doesn't need to use T_MIN=0.8 to reduce the number of models searched during optmize acq. This line is the reason for the FileNotFound exception. Please try to delete it and see if it still works.
Thanks.